k3s/token-helper.yaml

```diff
@@ -0,0 +1,242 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: token-helper
+  labels:
+    app.kubernetes.io/name: token-helper
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: token-helper
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: token-helper
+    spec:
+      serviceAccountName: default
+      containers:
+        - name: token-helper
+          imagePullPolicy: IfNotPresent
+          image: quay.io/wi_stefan/ishare-jwt-helper:0.3.0
```

Comment:
Because the others are framework components, that might be subject to change in matrix-tests, while this is just a helper for the test, not a part of the framework